hbase-server/src/test/java/org/apache/hadoop/hbase/master/TestClusterRestartFailover.java

```diff
@@ -131,10 +132,17 @@ public void test() throws Exception {
       .getServerNode(SERVER_FOR_TEST);
     assertNull("serverNode should be deleted after SCP finished", serverNode);
 
+    // this test has been flakey. When it is rerun by surefire, the underlying minicluster isn't
```

Comment:
Can not reset the metrics system in the AfterClass method or BeforeClass method?

Reply:
Posted a new patch that (I think) will reset the master metrics each time the mini cluster is started.